.gitignore

```diff
@@ -1,3 +1,4 @@
+files
```

Comment:
I added this as a symlink in my own git so I wouldn't have to keep going through inst/www/shared/... to access the JS and CSS files for testing.